src/modules/edc-demo/components/loading-state/loading-state.component.ts

```diff
@@ -0,0 +1,14 @@
+import {Component, HostBinding} from '@angular/core';
+
+@Component({
+  selector: 'edc-demo-loading-state',
+  templateUrl: './loading-state.component.html',
+})
+export class LoadingStateComponent {
+  @HostBinding('class.flex')
+  @HostBinding('class.flex-col')
+  @HostBinding('class.justify-center')
+  @HostBinding('class.items-center')
+  @HostBinding('class.min-h-[35rem]')
```

Comment:
This class for our layouting hack should be applied where the components are used, with class="min-h-[35rem]".